regcomp.c

```diff
@@ -6021,7 +6028,7 @@ S_regatom(pTHX_ RExC_state_t *pRExC_state, I32 *flagp, U32 depth)
                 char * e = RExC_end;
 
                 if (*s == 'g') {
-                    bool isrel = 0;
+                    bool isrel = FALSE;
```

Comment:
Why not use C99 'false' here and similarly below?  We already have instances of the newer in core; I see no need to be consistent; I think new stuff should be using the C names, just like we should be using `ptr_diff_t` instead of our macro that was just removed

Reply:
Because I didn't see `bool`/`true`/`false` on the list of approved C99 features in perlhacktips. On closer examination there is a "When is a bool not a bool?" section near the bottom, but technically it doesn't say we can rely on `true`/`false` being present.

More to the point, there are already about 240 instances of `TRUE`/`FALSE` in regcomp.c. I'd rather keep things locally consistent; a later patch can then lowercase all the boolean constants in the file in one go.

Reply:
fair enough